### PR TITLE
 New logging quickstart from Joel Tosi

### PR DESCRIPTION
**Pull request by sgilda**
This pull replaces Joel Tosi's pull 114: https://github.com/jboss-jdf/jboss-as-quickstart/pull/114

### CODE REVIEW

**Comment by sgilda** [comment]
@rafabene : They are generated for the markdown files, but I don't believe they are generated when we run the release-utils to generate the README.html files. Those files are checked into the jboss-eap quickstart repository and ship with the product. Since these quickstarts are merged to jboss-eap with minimal changes, I think the anchors need to be there for the generated README.html files to work.

**Comment by rafabene** [comment]
@sgilda  ah ok! Maybe we should work on the html generation to be consistent with Github Markdown

**Comment by sgilda** [comment]
@rafabene : That would be great! In the meantime, can we keep this one as is to be consistent with all the README files? They all use anchors.

**Comment by rafabene** [comment]
@sgilda I'll get the details with you on IRC.

File: logging/README.md [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+
+        * `server.log` - a standard log files produced by the application server
+        * `boot.log` - a standard log files produced by the application server
+
+
+
+Configure the Logging Quickstart Log File Handlers
+----------------------------------------------
+
+You will now add handlers to the server log properties file and configure the server to use them.
```
Comment (pmuir) [inline logging/README.md]: We seem to have switched into instructions in the future tense here.

Comment (sgilda) [inline logging/README.md]: Good catch!

**Comment by pmuir** [comment]
This looks excellent!

**Pull request merged**